source.py

```diff
@@ -8,21 +8,37 @@
 # ## Topic
 # *What problem are you (or your stakeholder) trying to address?*
 # 📝 <!-- Answer Below -->
+# The problem we are addressing is total energy consumption vs energy output
 
 # ## Project Question
 # *What specific question are you seeking to answer with this project?*
 # *This is not the same as the questions you ask to limit the scope of the project.*
 # 📝 <!-- Answer Below -->
+# On our trajectory (the world) how long can we really last with how we manage our energy. Is it really sustainable? Are there better decisions to be made regarding how to acquire green energy and reuse it.
```

Comment:
The question is well thought out and addresses the problem that the analysis can help provide an insight to.